testdata/compact_blocks.json

```diff
@@ -3,36 +3,36 @@
-    "compact": "10b4d5111a204ff234f7b51971cbeb7719a1c32d1c7e1ed92afafed266a7b1ae235717df050020c59ce0dc05"
+    "compact": "10b4d5111a204ff234f7b51971cbeb7719a1c32d1c7e1ed92afafed266a7b1ae235717df050028c59ce0dc05"
```

Comment:
What changed about these?

Reply:
Nevermind, the next commit replaces them anyway.

Reply:
The protobuf field indicator changed due to the backwards-incompatible protobuf specification change. It altered only one or two bytes in the format here, while the next commit then inserted the `prevHash` fields.